### PR TITLE
FOLIO-3179 ui-inventory-es: align dependency versions affected by Inventory's Optimistic Locking

### PR DESCRIPTION
**Pull request by MikhailFokanov-ebsco**
ui-inventory-es: align dependency versions affected by Inventory's Optimistic Locking

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## [6.4.0](https://github.com/folio-org/ui-inventory-es/tree/v6.4.0) (2021-06-17)
 [Full Changelog](https://github.com/folio-org/ui-inventory-es/compare/v6.3.0...v6.4.0)
 * Also support `circulation` `11.0`. Refs UISEES-65.
+* Replace okapiInterfaces dependency `item-storage: 8.0` by `inventory 11.0`. Refs FOLIO-3179.
```
Comment (julianladisch) [inline CHANGELOG.md]:
```suggestion
* Add additional inventory interfaces for optimistic locking: `inventory 11.0`, `instance-storage 8.0`, `holdings-storage 5.0`, `item-storage 9.0`. Refs FOLIO-3179.
```

**Comment by julianladisch** [comment]
The test/fixtures/instances.js doesn't contain the _version property.

Please make a note how you checked that the InstanceEdit page properly returns the _version property when updating the instance. Did you code inspection, or have you looked at the JSON sent to the back-end in the browser developer tools?

**Comment by julianladisch** [comment]
Or do you simply don't care whether InstanceEdit can still write an update because the code will be dropped anyhow?

**Comment by MikhailFokanov** [comment]
Yesterday I had a meeting with Zak. This UI module will be discounted and some components will be merged to Inventory UI. InstanceEdit will be definitely not merged as it is not touched and outdated in this module.

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**